### DISCUSSION TITLE
PCAP issue

### DISCUSSION THREAD

**Discussion by cyberkimh** [Security-Onion-Solutions/securityonion#4245] [Unsupported Versions]
Hi I was wondering if you have any advice as when I open an alert in Hunt or Kibana and try and view the pcap in PCAP I get a 404 error.
The nodes are all upto date and there's pcaps on the sensor.
Running /opt/so/log/sensoroni/sensoroni.log on the sensor shows: 
![image](https://user-images.githubusercontent.com/61547463/118845407-a74e6b80-b8c3-11eb-9b7c-0a637a622e14.png)

and running  /opt/so/log/soc/sensoroni-server.log on the management node shows:
![image](https://user-images.githubusercontent.com/61547463/118845928-1c21a580-b8c4-11eb-9f92-739934a882e5.png)

Any advice would be welcome thanks.

**Comment by cyberkimh** [comment]
Hi, just wondering if anyone has any suggestions to fix this, as we updated to 2.3.51 but we're still not able to view the pcaps in the console. Does RAM affect this as our management node has 16GB of RAM so we're wondering if that is the issue.

**Comment by jertel** [comment]
Can you paste a screenshot of the 404?

**Reply by cyberkimh** [reply]
![image](https://user-images.githubusercontent.com/61547463/119638657-c773b280-be0e-11eb-845f-4cf5ff582426.png)

**Reply by cyberkimh** [reply]
We open an alert in Hunt, click to open PCAP and it used to pivot to the PCAP tab, but now just stays on the alert's page and errors out. It doesn't even show 'pending' or 'failed to load' in the PCAP tab. 
This happens if we browse to the SOC and on the analyst VM. We're quite stuck.

**Reply by jertel** [reply]
Does it take a couple minutes before it errors out with the 404? From the screenshot above and the error you pasted in a second answer below, it appears that the Elastic server server is either not running, or is very slow to respond. If you search for the same document ID via a query like `_id:"PD3RpXkBYiVqzOR6aGQu"` in Kibana, does it find it quickly?

**Reply by cyberkimh** [reply]
It does take about 1 minute to error on the console, but just searched for it in Kibana and it found it very quickly.

**Reply by jertel** [reply]
You can see from the log line that the request to Elastic is only searching for the same "_id" criteria, so it's surprising that a request from SOC times out. I suggest tailing the elasticsearch log file and watching it the moment you click the PCAP quick action. You'll need to confirm that Elasticsearch is receiving that request first, and then if it is, review any additional information that elasticsearch might log with it.

**Comment by cyberkimh** [comment]
We're actually getting a new error on the management node: 
![image](https://user-images.githubusercontent.com/61547463/119690504-a8dadf00-be41-11eb-8fee-d7c2e6bf5231.png)

**Reply by Stephen-Ridgway** [reply]
This is an example of the error we see in the sensoroni-server log:
tail -f /opt/so/log/soc/sensoroni-server.log | grep AolAqXkBYiVqzOR6nfTX

timestamp=2021-05-27T10:18:46.13355298Z level=info message="Searching Elasticsearch" query="{\"query\" : { \"bool\": { \"must\": { \"match\" : { \"_id\" : \"AolAqXkBYiVqzOR6nfTX\" }}}}}"

timestamp=2021-05-27T10:20:46.136625326Z level=error message="Unable to lookup initial document record" error="net/http: timeout awaiting response headers" query="{\"query\" : { \"bool\": { \"must\": { \"match\" : { \"_id\" : \"AolAqXkBYiVqzOR6nfTX\" }}}}}"

**Reply by Stephen-Ridgway** [reply]
It takes exactly 2 mins to time out. Could anyone point me to the setting for that timeout?

**Reply by cyberkimh** [reply]
I thought it was 1 min to timeout but it's taking exactly 2 mins to timeout.

**Reply by Stephen-Ridgway** [reply]
After much digging, we've noticed that Curator isn't running. Could this be part of the problem?

If we try so-curator-start we get this error:
￼
![image](https://user-images.githubusercontent.com/80473697/119819676-894ac180-bee8-11eb-969d-42869b11021b.png)
